drivers/wifi/nxp/nxp_wifi_drv.c

```diff
@@ -1356,10 +1356,12 @@ static int nxp_wifi_set_twt(const struct device *dev, struct wifi_twt_params *pa
 		twt_setup_conf.implicit = params->setup.implicit;
 		twt_setup_conf.announced = params->setup.announce;
 		twt_setup_conf.trigger_enabled = params->setup.trigger;
+		twt_setup_conf.twt_info_disabled = params->setup.twt_info_disable;
```

Comment:
commit log says quick setup but these are added to the full TWT command

Reply:
Have separated the quick setup and I-TWT as independence PRs